UPGRADE.md

```diff
@@ -0,0 +1,46 @@
+# Upgrade Guide for SolarWinds APM PHP Library 9.x
+This document outlines the steps and considerations for upgrading from SolarWinds APM PHP Library version 8.x to 9.x.
+
+## Prerequisites
+- Ensure you have a backup of your current application and its dependencies.
+- Review the [releases](https://github.com/solarwinds/apm-php/releases)
+- Familiarize yourself with the [TROUBLESHOOTING.md](./TROUBLESHOOTING.md) for potential issues during the upgrade process.
+- Check compatibility of your PHP version with the new library version.
+- Review any custom instrumentation or integrations you have implemented.
+
+## API changes from 8.x to 9.x
+- The 8.x version used legacy internal hooks for instrumentation, while 9.x leverages OpenTelemetry auto-instrumentation. Review your custom instrumentation to ensure compatibility.
+
```

Comment:
Is the table below workslop? The checkmark / stop sign columns don't need to be there, and the Recommendation is simply wrong for things like mapping `solarwinds_apm_log*` to OTel Logging API.

Reply:
My bad. After re-visiting `solarwinds_apm_log*` code again, I should map `solarwinds_apm_log*` to OTel Trace API. Also removed the columns.